libimage/copier.go

```diff
@@ -206,13 +206,17 @@ func getDockerAuthConfig(name, passwd, creds, idToken string) (*types.DockerAuth
 	}
 }
 
+// NewCopier is a simple, exported wrapper for newCopier
+func NewCopier(options *CopyOptions, sc *types.SystemContext) (*copier, error) {
```

Comment:
Make the `copier` type public as well, so that it can be a struct field?

Reply:
i was trying to do minimal exposure here... but if the gang wants that, i can do it.

Reply:
Even if the type is private, external packages can call all public methods, so I don’t think making the type private helps encapsulate things.